core/src/main/java/org/elasticsearch/index/query/SimpleQueryStringBuilder.java

```diff
@@ -38,6 +39,7 @@
     private String queryName;
     private String minimumShouldMatch;
     private int flags = -1;
+    private float boost = 1.0f;
```

Comment:
I think we use -1 as default in master for all of the other queries, and we move to 1.0f in the query-refactoring branch. Should we use -1 here too for now?